DataFormats/PatCandidates/interface/PackedTriggerPrescales.h

```diff
@@ -14,9 +14,9 @@ namespace pat {
     ~PackedTriggerPrescales() {}
 
     // get prescale by index.
-    int getPrescaleForIndex(int index) const;
+    double getPrescaleForIndex(int index) const;
```

Comment:
do we wish to consider protecting this against people containing to call this expecting an int via the template with default to int and static assert for having an double/FracPrescale